scripts/ci-test.sh

```diff
@@ -69,24 +72,23 @@ oracletest() {
   npm i strong-oracle
   # Check for node_modules/strong-oracle explicitly because even if installation of it fails, npm doesn't catch it.
   if [[ ! -d node_modules/strong-oracle ]] ; then echo 'node_modules/strong-oracle is not there, return error.' ; exit 1 ; fi
-  # Build container
-  LEDGER_UNIT_DB_URI='oracle://system:oracle@localhost:49161/' LD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha
+  npm run setup-oracle
+  npm run test-oracle
 }
 
 oneNode() {
   lint
   dockerBuild
   sqlitetestest
   integrationtest
-  mysqltest
```

Comment:
Check constraints in MySQL is handled as a trigger before insert and would have to rely on a fast query (usually in row math is best where you already have the base and delta values to evaluate).   That trigger can be ported to any db, but  since you plan to have multiple scripts for each supported flavor of db, it would just be a different methodology for the same function/logic.